src/midiEditorCommands.cpp

```diff
@@ -740,7 +740,7 @@ void cmdMidiMoveCursor(Command* command) {
 		fakeFocus = FOCUS_NOTE;
 		s << " " << format(
 			translate_plural("{} note", "{} notes", count), count);
-		int mutedCount = count_if(notes.begin(), notes.end(), [](auto note) { return note.muted; });
+		int mutedCount = (int)count_if(notes.begin(), notes.end(), [](auto note) { return note.muted; });
```

Comment:
The correct fix for a lot of these is probably to change the type to size_t (or just auto) instead, unless we end up passing them to REAPER directly or indirectly, in which case casting to int is easier/more correct.